modules/xcm-bridge-hub/src/exporter.rs

```diff
@@ -0,0 +1,184 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
Here, I had to change the original `XcmBlobHaulerAdapter` a bit, because it doesn't work well with dynamic lanes. So it needs more careful review